lib/winapi/libloaderapi.c

```diff
@@ -37,7 +37,7 @@ static PIMAGE_EXPORT_DIRECTORY find_edataxb (void)
     PXBE_SECTION_HEADER section_header_addr = CURRENT_XBE_HEADER->PointerToSectionTable;
 
     for (DWORD i = 0; i < num_sections; i++) {
-        if (memcmp(section_header_addr[i].SectionName, ".edataxb", 8) == 0) {
+        if (strncmp(section_header_addr[i].SectionName, ".edata", 8) == 0) {
```

Comment:
Hm shouldn't that be a `strncmp(section_header_addr[i].SectionName, ".edata", 6)`? (Changing the size argument from 8 to 6)

Reply:
Was the intention to also support the old `.edataxb` style by only checking the prefix? Why? This wasn't mentioned in the PR description; behaviour is also a bit sneaky in the code (hard to spot that this might happen).

Reply:
> Hm shouldn't that be a `strncmp(section_header_addr[i].SectionName, ".edata", 6)`? (Changing the size argument from 8 to 6)

No, that would return a false positive for everything starting with `.edata`.

> Was the intention with strncmp to also support the old .edataxb style by only checking the prefix?

The opposite, I wanted to prevent that. However, while the above `strncmp` should work as expected, it's actually nonsense to do it that way, and the initial `memcmp` was incorrect as well - I somehow had it in my mind that the section names are null-terminated 8-byte fields, but they're not, the library names are. Section names are properly null-terminated strings, so a regular `strcmp` is the correct thing to do here.
I've updated the code to reflect that now.